osvr/RenderKit/RenderManager.h

```diff
@@ -1168,8 +1180,16 @@ namespace renderkit {
         bool hasHeadPose() const;
         bool getLastHeadPose(OSVR_TimeValue& tv, OSVR_Pose3& pose) const;
 
+		bool hasLeftEyePose() const;
+		bool hasRightEyePose() const;
+		bool getLastLeftEyePose(OSVR_TimeValue& tv, OSVR_Pose3& pose) const;
+		bool getLastRightEyePose(OSVR_TimeValue& tv, OSVR_Pose3& pose) const;
```

Comment:
Maybe rename these to hasLeftViewpointPose to reduce confusion with eye trackers, which we aren't using here (unless eye tracker plugins explicitly add viewpoint interfaces or aliases for them).